dbms/src/Functions/FunctionsString.cpp

```diff
@@ -4463,6 +4465,13 @@ class PadUTF8Impl : public IFunction
     }
 };
 
+static Int64 getResult(const ColumnString::Chars_t & chars, size_t size, size_t offset)
+{
+    if unlikely (size == 0)
```

Comment:
I think even if `size == 0`, `return chars[offset]` is ok because for empty string column, ColumnString will store `\0` in `chars`

Reply:
> I think even if `size == 0`, `return chars[offset]` is ok because for empty string column, ColumnString will store `\0` in `chars`

done